website/docs/docs/build/snapshots.md

```diff
@@ -512,6 +512,8 @@ Basically – keep your query as simple as possible! Some reasonable exceptions
 
 Snapshot <Term id="table">tables</Term> will be created as a clone of your source dataset, plus some additional meta-fields*.
 
+In v1.9 or with [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), these column names can be customized to your team or organizational conventions via the [`snapshot_meta_column_names`](/reference/resource-configs/snapshot_meta_column_names) config.
```

Comment:
"Starting in 1.9"?

Reply:
should we add a warning that since we don't handle migrations either use this config for new snapshots or update your column names manually in the warehouse first?

Reply:
The page that it links to has the following callout -- does that look like it meets the need?

```
:::note

To avoid any uninentional data modification, dbt will **not** automatically apply any column renames. So if a user applyies `snapshot_meta_column_names` config for a snapshot without updating the pre-existing table, they will get an error. Our recommendation is to either only use these settings for net-new snapshots, or to arrange an update of pre-existing tables prior to committing a column name change.

:::
```

Reply:
Alternatively we can update the error message to point to the new page? Maybe that's better...

Reply:
Okay! Easy to change our minds later and do a small follow-up PR to the docs too.